### PR TITLE
Adds confessional intercoms to Delta/Pubby, cleans up dvars and wrong freqs/settings

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Added private intercoms to the confession booths of the Deltastation+Pubbystation chapels.
fix: Fixed invalid radio frequencies on interrogation chamber/confession booth intercoms.
/:cl:

Delta/Pubby were missing intercoms inside the confession booth, I added them to both.

### CODE REVIEW

**Comment by SpaceManiac** [comment]
Even numbers are not valid radio frequencies

**Review by SpaceManiac — DISMISSED** [review]

**Comment by 81Denton** [comment]
Looks like most intercoms in general have dvars and some have invalid freqs as well. I'll open another PR for dvar cleanup later.

**Comment by Kagus** [comment]
Finally, a buff for chaplain!

**Comment by 81Denton** [comment]
Large commit, tl;dr
- Removed a bunch of dirty vars. Most intercoms had the default frequency and things like `broadcasting = FALSE`, `freerange = FALSE` and `listening = TRUE` defined without a need for it.
- Fixed invalid radio frequencies, especially interrogation intercoms which had them set to 1424.
- Removed `freerange = TRUE` on interrogation intercoms - this would allow the person using the intercom to listen to/broadcast on any channel, which is reserved for command/tcomms/AI.

**Comment by ExcessiveUseOfCobblestone** [comment]
I was under the impression the freq was intended since it couldn't be "hacked".

Basically a hand-wave for simulating communication between the confessional booth without making a unique asset :^) .

**Comment by SpaceManiac** [comment]
The intercom code just rounds the frequency up to 1481 on load.

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -4171,10 +4159,8 @@
 "ajO" = (
 /obj/structure/table/wood,
 /obj/item/radio/intercom{
-	broadcasting = 0;
 	dir = 8;
-	listening = 1;
-	name = "Station Intercom (Court)"
+
```
Comment (SpaceManiac) [inline _maps/map_files/BoxStation/BoxStation.dmm]: spurious blank line

File: code/game/objects/items/devices/radio/intercom.dm [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /obj/item/radio/intercom
-	name = "station intercom"
+	name = "Station Intercom (General)"
```
Comment (SpaceManiac) [inline code/game/objects/items/devices/radio/intercom.dm]: might ought to be `"\improper Station Intercom (General)"` or perhaps just take this opportunity to name station intercoms "station intercom" w/o the unnecessary capitalization

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/radio/intercom.dm (hunk 2 above)
Comment (81Denton) [reply code/game/objects/items/devices/radio/intercom.dm]: yeah, it does look better

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**